src/types.ts

```diff
@@ -67,7 +67,7 @@ export interface ModuleOptions {
   headings?: {
     anchorLinks?: {
       [heading in 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6']?: boolean
-    } | false
+    } | boolean
```

Comment:
issue: please see the type checking issue this PR and #206 introduced: https://github.com/nuxt-modules/mdc/issues/213